src/devices/ikea.ts

```diff
@@ -210,6 +211,7 @@ const definitions: DefinitionWithExtend[] = [
         vendor: 'IKEA',
         description: 'TRADFRI bulb B22, white spectrum, globe, opal, 1055 lm',
         extend: [addCustomClusterManuSpecificIkeaUnknown(), ikeaLight({colorTemp: true}), m.identify()],
+        toZigbee: [tz.ikea_bulb_unfreeze],
```

Comment:
This could specify different unfreeze mechanisms in the future (as in the [original code](https://github.com/Koenkk/zigbee2mqtt/issues/18574#issuecomment-1680923093)), but I only have this one bulb type to test with, so I'm just changing this for now

Reply:
Hardcoded unfreeze mechanism, see PR description for details